src/lib/dataviews/mode/DataViewLocal.ts

```diff
@@ -82,18 +86,26 @@ export class DataViewLocal extends DataViewMode {
   }
 }
 
-function createCategory(name: string, data: number[], operation: AggregationType) {
-  const numberFilter = function numberFilter(value: number | undefined) {
-    return Number.isFinite(value as number);
-  };
+function createCategory(name: string, data: number[] | number, operation: AggregationType) {
+  let categoryValue;
 
-  const filteredValues = data
-    .map(number => castToNumberOrUndefined(number))
-    .filter(numberFilter) as number[];
+  if (operation === AggregationType.COUNT) {
```

Comment:
I think we should call aggregate at all times in this code to make it standard given that we have the method already implemented.

So I would leave the method as is, but avoiding to filter values if operation is count.

```js
let categoryValues = data;
const shouldFilterValues = operation !== AggregationType.COUNT;

if (shouldFilterValues) {
  const numberFilter = function numberFilter(value: number | undefined) {
    return Number.isFinite(value as number);
  };

  categoryValues = (data as number[])
    .map(number => castToNumberOrUndefined(number))
    .filter(numberFilter) as number[];
}

return {
  name,
  value: aggregate(filteredValues, operation)
}
```